### ISSUE TITLE
Prevent narrowing of the content past a point that interferes with the display.

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#1371]
One of the testers had his screen fairly narrow and it caused display issues, particularly in the file upload section. See screenshot. It would be good to address this. My guess is that setting a minimum width somehow would be the way to go.

![MInimumWidth.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/b5040202-847d-4a4e-9230-a5af86f74c88)

**Comment by jcoyne** [comment]
Preventing a narrower screen may be an accessibility concern and might prevent the site from being useful when people use tablets or mobile devices.  A better solution would be to develop a design for this row for narrower viewports.

**Comment by amyehodge** [comment]
@astridu This isn't a high priority, but would benefit from your input when you have a chance. Thanks.

**Comment by astridu** [comment]
We probably should have some breakpoints, but wondering if the screen was narrow because she had the testing doc open side by side? Or is this a more widespread problem? I hadn't thought of it because I assumed people wouldn't be viewing on tablet/mobile, but if you think it could be a common use case where they reduce the size of their window because they're working side-by-side on another browser, we can revisit.

**Comment by amyehodge** [comment]
I have no idea why he had the screen so narrow, since I could see a lot of desktop on the right. Screenshot below.
![narrowscreen](https://user-images.githubusercontent.com/5558511/116086705-ee469980-a654-11eb-8346-0c4041a2fb3e.png)

I would not expect people to be doing this on a tablet or mobile, because they would need to have easy access to the files to upload as well. I have never had a user mention doing it on a mobile device.

**Comment by jcoyne** [comment]
@amyehodge There is no barrier to uploading from mobile devices.  Many people use tablet, or tablet like devices (I'm thinking Microsoft Surface) as their primary devices.

**Comment by amyehodge** [comment]
@jcoyne that's true for many things, but I find it less likely that people are doing their main research on mobile devices or even have files stored in places that are accessible via mobile devices. How easy is it to get to a department server, for example, via a mobile device? Then again, I remember being shocked when some previous survey of library users produced a comment from someone who complained of some task on our website being difficult to do from a mobile WHILE RIDING A BIKE. Is there a way we can find out how many people access sdr.stanford.edu from a mobile device?

**Comment by jcoyne** [comment]
I believe that information is available via google analytics. I don't have the account for that. 

Regardless, of what the usage data is now, the current sdr.stanford.edu works acceptably on all sizes. I don't think we should go out of our way to implement a design that would prevent people from using the site on the device of their choice.

**Comment by andrewjbtw** [comment]
I think there's a screen width/zoom level that results in the save/publish button disappearing in the current Hydrus. It's come up a few times when helping users but may have been addressed.

I would think a tablet is more likely than a phone as mobile use, but that even if people aren't doing the whole deposit on mobile they may want to review progress or check on previous deposits there. FWIW, I've remediated items from my phone before, but I would discourage generalizing from my own usage.
